doc/Rules.md

```diff
@@ -535,7 +535,7 @@ Code blocks are included in this rule by default since it is often a
 requirement for document readability, and tentatively compatible with code
 rules. Still, some languages do not lend themselves to short lines.
 
-Rationale: Extremely long lines can be difficult to work with in some editors.
+Rationale: Extremely long lines can be difficult to work with some editors.
```

Comment:
The "in" should not be removed. The "with" in "work with" does not refer to the editors, but to the long lines, so the "in" is still needed.